source/agg-pipeline-builder/create-a-view.txt

```diff
@@ -12,5 +12,44 @@ Create a View from Pipeline Results
    :depth: 1
    :class: singlecol
 
-TODO: 
+To quickly access the results of an aggregation pipeline without having to 
+execute an aggregation pipeline, you can create a :ref:`view <compass-views>` 
+on |compass|. Views are read-only, so they can help keep your data secure by 
+limiting user access to a predefined set of results. 
+
+About This Task 
+---------------
+
+Creating a view does not save the aggregation pipeline itself. 
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the Save dropdown button
```

Comment:
(options)
You could just write "Click Save" as the step title. 
If you do include drop-down, it should be hyphenated. 
Also, use either button or menu consistently. :) This last one is tricky, but I think I'd call it a button.